pennylane_forest/device.py

```diff
@@ -359,7 +359,7 @@ def mat_vec_product(self, mat, vec, wires):
 
         return np.reshape(state_multi_index, 2 ** self.num_wires)
 
-    def probability(self, wires=None):
+    def analytic_probability(self, wires=None):
```

Comment:
This change follows-up the change in the structure of QubitDevice:
XanaduAI/pennylane#573